fuel-tx/Cargo.toml

```diff
@@ -14,17 +14,22 @@ description = "FuelVM transaction."
 bitflags = { workspace = true }
 derivative = { version = "2.2.0", default-features = false, features = ["use_core"], optional = true }
 derive_more = { version = "0.99", default-features = false, features = ["display"] }
+duplicate = "1.0"
 fuel-asm = { workspace = true, default-features = false }
 fuel-crypto = { workspace = true, default-features = false }
 fuel-merkle = { workspace = true, default-features = false, optional = true }
 fuel-types = { workspace = true, default-features = false }
+getrandom = { version = "0.2", optional = true }
 hashbrown = { version = "0.14", optional = true }
 itertools = { version = "0.10", default-features = false, optional = true }
+js-sys = { version = "0.3", optional = true }
```

Comment:
`js-sys` is used to return proper js error types, for instance here:

https://github.com/FuelLabs/fuel-vm/blob/812411437a37b06e56759c27449b792c7d27dc01/fuel-tx/src/tx_pointer.rs#L117-L121

Reply:
Why do we return `JsValue` in one cases and `js_sys::Error` in another?

Reply:
I had missed those since I searched for `JsValue>` to find use in `Result` types, but I had left inline comments so that it looked like this `Result<..., JsValue /* Comment */>` instead.

Fixed in 10ac139f.